cranelift/fuzzgen/src/function_generator.rs

```diff
@@ -457,6 +457,125 @@ fn valid_for_target(triple: &Triple, op: Opcode, args: &[Type], rets: &[Type]) -
             )
         }
 
+        Architecture::Riscv64(_) => {
+            exceptions!(
+                // TODO
+                (Opcode::IaddCout),
+                // TODO
+                (Opcode::Udiv, &[I128, I128]),
+                // TODO
+                (Opcode::Sdiv, &[I128, I128]),
+                // TODO
+                (Opcode::Urem, &[I128, I128]),
+                // TODO
+                (Opcode::Srem, &[I128, I128]),
+                // TODO
+                (Opcode::Iabs, &[I128]),
+                // https://github.com/bytecodealliance/wasmtime/issues/5497
+                (Opcode::Smin, &[I128, I128]),
+                (Opcode::Umin, &[I128, I128]),
+                (Opcode::Smax, &[I128, I128]),
+                (Opcode::Umax, &[I128, I128]),
+                // TODO
+                (Opcode::Bitselect, &[I128, I128, I128]),
+                // TODO
+                (Opcode::Bswap),
+                // https://github.com/bytecodealliance/wasmtime/issues/5523
+                (Opcode::Rotl, &[I128, I8]),
+                (Opcode::Rotl, &[I128, I16]),
+                (Opcode::Rotl, &[I128, I32]),
+                (Opcode::Rotl, &[I128, I64]),
+                (Opcode::Rotl, &[I128, I128]),
+                // https://github.com/bytecodealliance/wasmtime/issues/5525
+                (Opcode::Sextend, &[I8], &[I128]),
+                (Opcode::Sextend, &[I16], &[I128]),
+                (Opcode::Sextend, &[I32], &[I128]),
+                (Opcode::Sextend, &[I64], &[I128]),
+                (Opcode::Sextend, &[I128], &[I128]),
```

Comment:
As an observation for any of us working on this list in the future, exceptions like these can be abbreviated by using more wildcards:

```suggestion
                // https://github.com/bytecodealliance/wasmtime/issues/5523
                (Opcode::Rotl, &[I128, _]),
                // https://github.com/bytecodealliance/wasmtime/issues/5525
                (Opcode::Sextend, _, &[I128]),
```

I don't think that's important, I just wanted to mention it while merging this.

Reply:
Oh, I didn't know that! That's really nice!